### PR TITLE
[Term Entry]:  add new doc entry for the python .translate() method.

### LINKED ISSUE #3366: [Term Entry] Python Strings .translate()
### Reason/inspiration (optional)

We would like a new term entry on the `.translate()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/translate/translate.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by adeyinkaezra123**
### Description

Added a term entry for the `.translate()` method.  The method returns a copy of the string in which each character has been mapped through the given translation table replacing or removing specified characters.

Closes #3366 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/translate/translate.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  # translation table - a dictionary
+  # If a character is mapped to None, the character is deleted from the return string
+  translation = {97: None, 98: None, 99: 105}
+
+  string = "abcdef"
+  print("Original string:", string)
+
+  # translates string
+  print("Translated string:", string.translate(translation))
+  # Returns: idef
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/translate/translate.md:64]: Again, show the output in its own ` ```shell` block after the example.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@adeyinkaezra123 I did a first review and left you a few comments.

**Comment by adeyinkaezra123** [comment]
Made the necessary changes 👍

File: content/python/concepts/strings/terms/translate/translate.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  # translates original_string by mapping 'a' to '1', 'b' to '2', and 'c' to '3'
+  translated_string = original_string.translate(translation_table)
+
+  print("Translated string:", translated_string)
+  ```
+  
+  **Return value**
+	```shell
+	>	123def
+	```
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/translate/translate.md:49]:
````suggestion
This example outputs the following:

```shell
Original string: abcdef
Translated string: 123def
```
````

File: content/python/concepts/strings/terms/translate/translate.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  string = "abcdef"
+  print("Original string:", string)
+
+  # translates string
+  print("Translated string:", string.translate(translation))
+  ```
+  **Return value**
+	```shell
+	>	idef
+	```
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/translate/translate.md:67]:
````suggestion
This example results in the following output:

```shell
Original string: abcdef
Translated string: idef
```
````

File: content/python/concepts/strings/terms/translate/translate.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  original_string = "abcdef"
+  print("Original string:", original_string)
+
+  translation_table = str.maketrans('abc', '123')
+
+  # translates original_string by mapping 'a' to '1', 'b' to '2', and 'c' to '3'
+  translated_string = original_string.translate(translation_table)
+
+  print("Translated string:", translated_string)
+  ```
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/translate/translate.md:44]:
````suggestion
```py
original_string = "abcdef"
print("Original string:", original_string)

translation_table = str.maketrans('abc', '123')

# translates original_string by mapping 'a' to '1', 'b' to '2', and 'c' to '3'
translated_string = original_string.translate(translation_table)

print("Translated string:", translated_string)
```
````

File: content/python/concepts/strings/terms/translate/translate.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  # translation table - a dictionary
+  # If a character is mapped to None, the character is deleted from the return string
+  translation = {97: None, 98: None, 99: 105}
+
+  string = "abcdef"
+  print("Original string:", string)
+
+  # translates string
+  print("Translated string:", string.translate(translation))
+  ```
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/translate/translate.md:63]:
````suggestion
```py
# translation table - a dictionary
# If a character is mapped to None, the character is deleted from the return string
translation = {97: None, 98: None, 99: 105}

string = "abcdef"
print("Original string:", string)

# translates string
print("Translated string:", string.translate(translation))
```

````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@adeyinkaezra123 I left some more comments.

File: content/python/concepts/strings/terms/translate/translate.md (hunk 5 above)
Comment (adeyinkaezra123) [reply content/python/concepts/strings/terms/translate/translate.md:65]: What of the codebyte example requirement stated in the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md#codebyte-example-if-applicable)?

Comment (SSwiniarski) [reply content/python/concepts/strings/terms/translate/translate.md:65]: @adeyinkaezra123 You can make it a codebyte, but in that case, remove the output as the user will see the output when running the codebyte.

Comment (adeyinkaezra123) [reply content/python/concepts/strings/terms/translate/translate.md:65]: Done 💯

**Review by SSwiniarski — APPROVED** [review]
@adeyinkaezra123 I made a few small edits now it looks good for a second review.

File: content/python/concepts/strings/terms/translate/translate.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+  - 'paths/computer-science'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:14]:
```suggestion
```

File: content/python/concepts/strings/terms/translate/translate.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+The **`.translate()`** method takes a translation table as an argument, which can be created using the static method `maketrans()`. The translation table maps the ordinal of a character to its corresponding replacement or deletion.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:17]:
```suggestion
The **`.translate()`** method takes a translation table as an argument and maps the ordinal of a character to its corresponding replacement or deletion. A translation table can be created using the static method `maketrans()`. 
```

File: content/python/concepts/strings/terms/translate/translate.md (hunk 6 above)
Comment (caupolicandiaz) [reply content/python/concepts/strings/terms/translate/translate.md:14]: catalog content is limited to two elements

File: content/python/concepts/strings/terms/translate/translate.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+## Syntax
+
+The syntax of the `translate()` method is:
+
+```pseudo
+string.translate(table)
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:24]:
```suggestion
a_string.translate(table)
```

File: content/python/concepts/strings/terms/translate/translate.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+  - 'paths/computer-science'
+---
+
+The **`.translate()`** method takes a translation table as an argument, which can be created using the static method `maketrans()`. The translation table maps the ordinal of a character to its corresponding replacement or deletion.
+
+## Syntax
+
+The syntax of the `translate()` method is:
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:21]:
```suggestion
The syntax of the `.translate()` method is:
```

File: content/python/concepts/strings/terms/translate/translate.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+The following example shows custom character mapping using a translation table and `.translate()`:
+
+```py
+original_string = "abcdef"
+print("Original string:", original_string)
+
+translation_table = str.maketrans('abc', '123')
+
+# translates original_string by mapping 'a' to '1', 'b' to '2', and 'c' to '3'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:40]:
```suggestion
# Translates original_string by mapping 'a' to '1', 'b' to '2', and 'c' to '3'
```

File: content/python/concepts/strings/terms/translate/translate.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+Original string: abcdef
+Translated string: 123def
+```
+
+## Codebyte Example
+
+The following example shows how to delete characters using a manual translation table:
+
+```codebyte/python
+# translation table - a dictionary
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:58]:
```suggestion
# Translation table - a dictionary
```

File: content/python/concepts/strings/terms/translate/translate.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+# If a character is mapped to None, the character is deleted from the return string
+translation = {97: None, 98: None, 99: 105}
+
+string = "abcdef"
+print("Original string:", string)
+
+# translates string
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/translate/translate.md:65]:
```suggestion
# Translates string
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@adeyinkaezra123 I've completed my initial review. Please update the entry, and tag me when  you've finished making changes. Thanks

**Comment by adeyinkaezra123** [comment]
Committed all required changes @caupolicandiaz 💯

**Pull request merged**